src/util/getInMemoryPage.js

```diff
@@ -0,0 +1,41 @@
+const sorter = id => (a, b) => {
+  var nameA = a[id]
+  var nameB = b[id]
+  if (typeof nameA === 'string') nameA.toUpperCase()
```

Comment:
You need to store the return value into the original object to keep it uppercase